symfony/test/functional/tools/plugins/orangehrmFunctionalTestingPlugin/Controller/DeleteSavepointController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\FunctionalTesting\Controller;
+
+use OrangeHRM\Core\Controller\PublicControllerInterface;
+use OrangeHRM\Framework\Http\Request;
+use OrangeHRM\Framework\Http\Response;
+
+class DeleteSavepointController extends AbstractController implements PublicControllerInterface
+{
+    /**
+     * @param Request $request
+     * @return Response
+     */
+    public function handle(Request $request): Response
+    {
+        $savepointNames = $request->request->get('savepointNames', []);
+        $response = $this->getResponse();
+        $response->setContent(
+            json_encode(['success' => $this->getDatabaseBackupService()->deleteSavepoints($savepointNames)])
```

Comment:
Known issue there in savepoint deletion.
let set if we have two savepoints called `jobTitle`, `jobTitles`. If we delete only `jobTitle` then `jobTitles` also delete. Because both savepoint have same start string pattern `jobTitle`.
Mainly this deletion required when start test suite again and again. If we can add delete all savepoints in a Cypress hook, problem will solve.

Reply:
As we discussed [this commit](https://github.com/orangehrm/orangehrm/pull/1056/commits/5b76e59a4e993ab42cd8671062f14d6623073d05) will resolve this.